documentation/index.rst

```diff
@@ -252,11 +252,22 @@ functions and methods is the stdlib :mod:`py3:inspect` module.
    aliased to :class:`py3:object`.)
 
 
+.. function:: update_wrapper(wrapper, wrapped, assigned=functools.WRAPPER_ASSIGNMENTS, updated=functools.WRAPPER_UPDATES)
```

Comment:
Why do we want to expose this?

Reply:
This PR is about adding support for `__wrapped__` attribute, and there are two functions that need to be backported to provide the missing attribute: `update_wrapper` and `wraps` (that uses `update_wrapper` internally).

That said, the most frequently used one is `wraps`, so we can make the backported `update_wrapper` internal if you like.

Reply:
Ok, done!